client/app/docs.html

```diff
@@ -8,25 +8,34 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
 
     <link rel="stylesheet" href="styles/css/bootstrap.css">
+
+    <!-- build:css styles/css/docs.css -->
     <link rel="stylesheet" href="styles/css/docs.css">
+    <!-- endbuild -->
 
   </head>
   <body>
 
     <div sd-docs></div>
-
+    
+    <!-- build:js scripts/vendor-docs.js -->
     <script src="scripts/bower_components/jquery/dist/jquery.js"></script>
     <script src="scripts/bower_components/jquery-ui/jquery-ui.js"></script>
     <script src="scripts/bower_components/angular/angular.js"></script>
     <script src="scripts/bower_components/lodash/lodash.js"></script>
     <script src="http://google-code-prettify.googlecode.com/svn/trunk/src/prettify.js"></script>
     <script src="scripts/bower_components/requirejs/require.js"></script>
-
+    <!-- endbuild -->
+    
+    <!-- build:js scripts/superdesk-docs-core.js -->
     <script src="scripts/superdesk/services/asset.js"></script>
-
-    <script src="scripts/config.js"></script>
-
     <script src="docs/module.js"></script>
+    <!-- endbuild -->
+
+    <!-- build:js scripts/superdesk-docs-main.js -->
+    <!-- endbuild -->
```

Comment:
no, because this is the way of making empty `superdesk-docs-main.js` which I need for `requirejs` task